packages/angular-material/src/other/master-detail/master.ts

```diff
@@ -156,7 +157,6 @@ export class MasterListComponent
   selectedItemIdx: number;
   addItem: (path: string, value: any) => () => void;
   removeItems: (path: string, toDelete: number[]) => () => void;
-  propsPath: string;
```

Comment:
When building the library, the bundler doesn't seem to complain. But when testing, an error occurs because it already exists in [JsonFormsAbstractControl](https://github.com/eclipsesource/jsonforms/blob/master/packages/angular/src/abstract-control.ts).